src/services/compare/handlers/compare.js

```diff
@@ -0,0 +1,46 @@
+import { DynamoDBClient, GetItemCommand } from "@aws-sdk/client-dynamodb";
+const client = new DynamoDBClient({
+  region: process.env.region,
+});
+
+exports.handler = async function (event, context, callback) {
+  console.log("Received event:", JSON.stringify(event, null, 2));
+  const id = event.Context.Execution.Input.id;
+  var response = {
+    statusCode: 200,
+    match: false,
+  };
+  try {
+    const mmdlItem = await getItem(process.env.mmdlTableName, id);
+    const seatoolItem = await getItem(process.env.seatoolTableName, id);
```

Comment:
Should we maybe do a check to make sure these environment variables are defined. I mean technically we don't have to because it's in a try catch. Just a thought though

Reply:
i think because its in this handler if its not defined the deploy will fail